test/plots/areaPlotTests.ts

```diff
@@ -2,162 +2,177 @@
+        dataWithNaN[2] = { x: 0.4, y: NaN };
+        simpleDataset.data(dataWithNaN);
 
-      let areaPathString = TestMethods.normalizePath(areaPath.attr("d"));
-      TestMethods.assertAreaPathCloseTo(areaPathString, expectedPath, 0.1, "area d was set correctly (y=NaN case)");
+        let pointsInArea = getPointsInArea(areaPlot, dataWithNaN.slice(0, 2)).concat(getPointsInArea(areaPlot, dataWithNaN.slice(3, 5)));
+        TestMethods.assertLinePathEqualToDataPoints(areaPath.attr("d"), pointsInArea, xScale, yScale);
 
-      dataWithNaN[2] = { foo: NaN, bar: 0.4 };
-      simpleDataset.data(dataWithNaN);
+        dataWithNaN[2] = { x: NaN, y: 0.4 };
+        simpleDataset.data(dataWithNaN);
 
-      areaPathString = TestMethods.normalizePath(areaPath.attr("d"));
-      TestMethods.assertAreaPathCloseTo(areaPathString, expectedPath, 0.1, "area d was set correctly (x=NaN case)");
+        TestMethods.assertLinePathEqualToDataPoints(areaPath.attr("d"), pointsInArea, xScale, yScale);
 
-      let dataWithUndefined = areaData.slice();
-      dataWithUndefined[2] = { foo: 0.4, bar: undefined };
-      simpleDataset.data(dataWithUndefined);
+        let dataWithUndefined = areaData.slice();
+        dataWithUndefined[2] = { x: 0.4, y: undefined };
+        simpleDataset.data(dataWithUndefined);
 
-      areaPathString = TestMethods.normalizePath(areaPath.attr("d"));
-      TestMethods.assertAreaPathCloseTo(areaPathString, expectedPath, 0.1, "area d was set correctly (y=undefined case)");
+        TestMethods.assertLinePathEqualToDataPoints(areaPath.attr("d"), pointsInArea, xScale, yScale);
 
-      dataWithUndefined[2] = { foo: undefined, bar: 0.4 };
-      simpleDataset.data(dataWithUndefined);
+        dataWithUndefined[2] = { x: undefined, y: 0.4 };
+        simpleDataset.data(dataWithUndefined);
 
-      areaPathString = TestMethods.normalizePath(areaPath.attr("d"));
-      TestMethods.assertAreaPathCloseTo(areaPathString, expectedPath, 0.1, "area d was set correctly (x=undefined case)");
+        TestMethods.assertLinePathEqualToDataPoints(areaPath.attr("d"), pointsInArea, xScale, yScale);
 
-      svg.remove();
+        svg.remove();
+      });
+
+      it("retains original classes when class is projected", () => {
+        let newClassProjector = () => "pink";
+        areaPlot.attr("class", newClassProjector);
+        areaPlot.renderTo(svg);
+        let areaPath = areaPlot.content().select(".area");
+        assert.isTrue(areaPath.classed("pink"));
+        assert.isTrue(areaPath.classed("area"));
+        svg.remove();
+      });
     });
 
-    describe("selections()", () => {
+    describe("Selections", () => {
+      let svg: d3.Selection<void>;
+      let xScale: Plottable.Scales.Linear;
+      let yScale: Plottable.Scales.Linear;
+      let twoPointData = [{x: 0, y: 0}, {x: 1, y: 1}];
+      let simpleDataset: Plottable.Dataset;
+      let areaPlot: Plottable.Plots.Area<number>;
+      let renderArea: d3.Selection<void>;
```

Comment:
I believe `twoPointData` and `simpleDataset` and `renderArea` are unneeded variables here